cmd/antrea-agent/agent.go

```diff
@@ -192,6 +196,26 @@ func run(o *Options) error {
 	}
 	go apiServer.Run(stopCh)
 
+	// Initialize flow exporter; start go routines to poll conntrack flows and export IPFIX flow records
+	if o.flowCollector != nil && features.DefaultFeatureGate.Enabled(features.FlowExporter) {
+		var connTrackDumper connections.ConnTrackDumper
+		if o.config.OVSDatapathType == ovsconfig.OVSDatapathSystem {
+			connTrackDumper = connections.NewConnTrackDumper(connections.NewConnTrackSystem(), nodeConfig, serviceCIDRNet, o.config.OVSDatapathType, agentQuerier.GetOVSCtlClient())
+		} else if o.config.OVSDatapathType == ovsconfig.OVSDatapathNetdev {
+			connTrackDumper = connections.NewConnTrackDumper(connections.NewConnTrackNetdev(), nodeConfig, serviceCIDRNet, o.config.OVSDatapathType, agentQuerier.GetOVSCtlClient())
+		}
+		connStore := connections.NewConnectionStore(connTrackDumper, ifaceStore, o.pollingInterval)
+		flowRecords := flowrecords.NewFlowRecords(connStore)
+		flowExporter, err := exporter.InitFlowExporter(o.flowCollector, flowRecords, o.exportInterval)
+		if err != nil {
+			// If flow exporter cannot be initialized, then Antrea agent does not exit; only error is logged.
```

Comment:
>Reminder.
Hi Jianjun, I think you were looking at old changes. Took care of that comment. Please select "all commits" or commit 8c23c174946e.

Reply:
Ok. My fault.